experimental/sql/test/lit.cfg

```diff
@@ -22,6 +22,3 @@ config.environment[
     "PATH"] = config.test_source_root + "/../tools/:" + os.environ["PATH"]
 
 config.available_features = []
-
-if sys.version_info[0] == 3 and sys.version_info[1] == 8:
-  config.available_features.append('python38')
```

Comment:
This change refers to the [MLIRConverter PR](https://github.com/google/iree-llvm-sandbox/pull/494/files/82b798c3d94765f160cda62d9b3e607f0668a58f#diff-ec2f94870e3e365cbdc19eacf297b7bbbf3516e10f201a988c4abb03b6c7abda). I realised that the same unnecessary part is included in this lit.cfg, so I removed it.